data/categories.txt

```diff
@@ -1469,7 +1477,7 @@ sw:Ununuzi maduka
 fa:مرکز خرید
 
 shop-department_store|@shopping
-en:Department Store|U+1F3EC|U+1F3EA|shopping|shopping centre|shopping arcade
```

Comment:
These shopping synonyms are for more general "shopping" search queries. Do you think that people don't expect to find a Department Store when they type "shopping"?

Reply:
"A [department store](https://wiki.openstreetmap.org/wiki/Tag:shop=department_store) is a large store with multiple clothing and other general merchandise departments." - I would not expect a department store to appear if I searched for a shopping centre. A [shopping centre](https://wiki.openstreetmap.org/wiki/Tag:shop=mall) is lots of shops, not just one: "a single building with multiple commercial retail stores where the entrances to the stores let onto a common enclosed area".

"Shopping" is a bit vague here, but either it appears as a synonym for _all_ shops, or it doesn't. As it is, it appears as a synonym for nothing else other than a department store, which is inconsistent.

A search for "shopping centre" should not return a department store. Surely?